src/shared/routes.js

```diff
@@ -3,16 +3,38 @@ import Root from './containers/root';
 import JoinUs from './containers/join-us';
 import Job from './containers/job';
 import Events from './containers/events';
+import AddEvent from './containers/add-event';
 import Event from './containers/event';
 import { Route } from 'react-router';
 import HttpError from './util/http-error';
+import request from 'superagent';
 import './containers/error';
 
+function authenticate (nextState, replaceState, callback) {
+  if (typeof window !== 'undefined') {
```

Comment:
not sure if we care - API will deny new event creation, but it would be still nice not to let anyone access the page.